src/wolfkrow/core/engine/task_graph.py

```diff
@@ -239,14 +222,15 @@ def execute_local(
                 continue
 
             ready = True
-            for dependencyName in task['obj'].dependencies:
+            for dependencyName in task.task.dependencies:
+                #TODO: Add dependency inheritance support + corrected logic around task name prefixes
                 # If the dependency has no entry, it means it was never actually added it to the task graph.
                 if results.get(dependencyName) is False:
                     ready = False
                     break
 
             if not ready:
-                logging.warning("This task's dependencies failed to execute. Skipping task: '%s'" % task['obj'].name)
+                logging.warning("This task's dependencies failed to execute. Skipping task: '%s'" % task.task.task_name)
```

Comment:
Task is a bad name here. Should be called `task_export` similar to the Deadline export method.